product-listing/products.css

```diff
@@ -1,3 +1,19 @@
+.navbar{
+    padding: 0.25rem;
+ }
+ .search-container{
+    flex-basis: 25%;
+    border-radius: 1.5rem;
+    background: var(--white)
+ } 
+.search-bar{
+    outline: none;
+    width: 100%;
+    border: none;
+    font-size: 1rem;
+    padding: 0.6rem 0.8rem;
+    margin: 0.5rem;
+}
 .middleSection{
     margin-top: 80px;
```

Comment:
maintain consistent units just suggestion

Reply:
I try to use rem for smaller values and px for bigger one? is it good approach? also can i use rem for bigger values as well?